docs/blog/2018-11-07-gatsby-for-apps/index.md

```diff
@@ -0,0 +1,249 @@
+
+The central idea I want to clarify is that typically app-like features are not only _possible_ with Gatsby, but intuitive and easy to implement due to the dynamic runtime enabled in every Gatsby application. It's _just_ React.
+
+Gatsby is for building dynamic, web applications, just as it is for building static sites. At this point, this has become clear. Gatsby can be used for many of the traditional cases for web applications, including authentication, client-only routes, dynamic data fetching, and more.
+
+However, what hasn't been made clear is _why_ you'd reach for Gatsby as opposed to other solutions.
+
+## _Why_ Gatsby for Apps?
+
+If we revisit some of the benefits that Gatsby provides, minimally:
+
+- Static rendering of React components and co-located data to static HTML
+- Optimizing data, images, etc. for blazing-fast performance
+- Internalizing performance patterns and best practices like [PRPL][prpl], route-based code splitting, etc.
+
+Each of these are worthy of far more detail, but suffice to say these are _great_ functionalities that you want not only in your _static_ site, but also in your application.
+
+These performance optimizations aren't opt-in; they're enabled, by default. As new performance techniques and optimizations gain popularity, we can internalize these just as we have for these others optimizations. These optimizations can then be made available to your end users merely by upgrading your version of Gatsby, in much the same way tooling improvements are available by upgrading [create-react-app][create-react-app].
+
+### Plugins and the Gatsby Ecosystem
+
+One of the key benefits of Gatsby is its highly modular architecture. Need a plugin for [sourcing data from Wordpress][gatsby-source-wordpress]? Sure, seems reasonable. Need to [transform yaml data][gatsby-transformer-yaml] into a usable, JavaScript object? Yeah, why not! Want to [stitch in a remote GraphQL API][gatsby-source-graphql] and inject the data at _build_ time? Oh, you're fancy! Want to load optimized, responsive, blur-in images? Yep.
+
+Let's take a look at that image functionality provided by one of our components, `gatsby-image`, in slightly more detail.
+
+#### `gatsby-image`
+
+Certainly one of my favorite components that Gatsby provides is `gatsby-image`. This component that we provide and maintain gives excellent image rendering capabilities, along with image optimizations enabled by plugins like [`gatsby-plugin-sharp`][gatsby-plugin-sharp]. These two techniques, along with the GraphQL API made available to any Gatsby application, vastly simplifies the developer experience of serving optimized images, providing a number of features, including:
```

Comment:
> This component that we provide and maintain gives excellent image rendering capabilities,

Already established that Gatsby provides it. Change first sentence to:

> Certainly one of my favorite components that Gatsby provides *and maintains* is `gatsby-image`.

Change second sentence to:

> This component ~that we provide and maintain~ gives excellent image rendering capabilities[...]